releases/release-5.3.1.md

```diff
@@ -0,0 +1,215 @@
+    - Increase the size of write batch for raftlog GC to speed up GC. [#11404](https://github.com/tikv/tikv/issues/11404)
+    (dup) - Fix a bug that TiKV cannot delete a range of data (`unsafe_destroy_range` cannot be executed) when the GC worker is busy [#11903](https://github.com/tikv/tikv/issues/11903)
+    - Fix a potential panic (#11746) when snapshot files have been deleted but the peer's status is still Applying. [#11746](https://github.com/tikv/tikv/issues/11746)
+    (dup) - Fix the issue that destroying a peer might cause high latency [#10210](https://github.com/tikv/tikv/issues/10210)
+    - Fix possible QPS drop when `level0_slowdown_trigger` is set explicitly with flow control enabled. [#11424](https://github.com/tikv/tikv/issues/11424)
+    - release-note [#11805](https://github.com/tikv/tikv/pull/11805)
+    (dup) - Fix a bug that the `any_value` function returns a wrong result when regions are empty [#11735](https://github.com/tikv/tikv/issues/11735)
+    (dup) - Update the proc filesystem (procfs) to v0.12.0 [#11702](https://github.com/tikv/tikv/issues/11702)
+    (dup) - Fix the issue that deleting an uninitialized replica might cause an old replica to be recreated [#10533](https://github.com/tikv/tikv/issues/10533)
+    - Please add a release note.None [#11632](https://github.com/tikv/tikv/pull/11632)
+    - Please add a release note. [#11616](https://github.com/tikv/tikv/pull/11616)
+    - Fix panic when cgroup controller is not mounted [#11569](https://github.com/tikv/tikv/issues/11569)
+    (dup) - Fix the metadata corruption issue when `Prepare Merge` is triggered after a new election is finished but the isolated peer is not informed [#11526](https://github.com/tikv/tikv/issues/11526)
+    (dup) - Fix the deadlock issue that happens occasionally when coroutines run too fast [#11549](https://github.com/tikv/tikv/issues/11549)
+    (dup) - Fix the issue that a down TiKV node causes the resolved timestamp to lag [#11351](https://github.com/tikv/tikv/issues/11351)
+    (dup) - Fix the issue that batch messages are too large in Raft client implementation [#9714](https://github.com/tikv/tikv/issues/9714)
+    (dup) - Fix a panic issue that occurs when Region merge, ConfChange, and Snapshot happen at the same time in extreme conditions [#11475](https://github.com/tikv/tikv/issues/11475)
+    (dup) - Fix the issue that TiKV cannot detect the memory lock when TiKV perform a reverse table scan [#11440](https://github.com/tikv/tikv/issues/11440)
+    (dup) - Fix the issue that RocksDB flush or compaction causes panic when the disk capacity is full [#11224](https://github.com/tikv/tikv/issues/11224)
+    (dup) - Fix a bug that tikv-ctl cannot return the correct Region-related information [#11393](https://github.com/tikv/tikv/issues/11393)
+    (dup) - Fix the issue that the average latency of the by-instance gRPC requests is inaccurate in TiKV metrics [#11299](https://github.com/tikv/tikv/issues/11299)
+
++ TiFlash
```

Comment:
TiFlash
bugfix

    - Fix the problem that `cast(input as decimal(x,y))` return wrong result when input overflows the range of `decimal(x,y)`
    - Fix the problem of TiFlash crashing when `max_memory_usage` and `max_memory_usage_for_all_queries` are enabled
    - Fix the problem that `cast(string ad real)` returns wrong result
    - Fixed the problem that cast(string as decimal) returns wrong result
    - Fix potential data inconsistency after altering a primary key column to a larger int data type
    - Fix the problem that `select (arg0, arg1) in (x,y)` returns wrong result
    - Fix the problem that 
    - Fix the problem that tiflash randomly crash when a mpp query is killed
    - Fix the problem that str_to_date return wrong result when the input argument has leading zeros.
    - Fix the problem that query gets wrong results when the filter is like `where <string>`.
    - Fix the problem that `cast(string as datetime)` return wrong result when the string if of format `%Y-%m-%d\n%H:%i:%s`.